README.md

```diff
@@ -24,6 +24,37 @@ This repository contains a curated list of awesome open source libraries that wi
 | [🔠 Industry-strength Natural Language Processing](#industry-strength-nlp) | [🍕 Industry-strength Reinforcement Learning](#industry-strength-rl) | [📊 Industry-strength Visualisation](#industry-strength-visualisation) |
 | [🙌 Industry-strength Recommender System](#industry-strength-recsys) | [📈 Industry-strength Evaluation](#industry-strength-evaluation) | [💰 Commercial Platform](#commercial-platform) |
 
+## Contributing to the list
+
+Thank you for taking the time to contribute to the awesome production machine learning list!
```

Comment:
This section is good, instead of copying the CONTRIBUTING.md let's link it here (i.e. please review our CONTRIBUTING.md requirements when submitting a PR).